controllers/machine_controller.go

```diff
@@ -271,7 +271,7 @@ func (r *MachineReconciler) reconcile(ctx context.Context, cluster *clusterv1.Cl
 	phases := []func(context.Context, *clusterv1.Cluster, *clusterv1.Machine) (ctrl.Result, error){
 		r.reconcileBootstrap,
 		r.reconcileInfrastructure,
-		r.reconcileNodeRef,
```

Comment:
Why are we removing this call but not the code?

/hold

Reply:
There is a name change, new name is `reconcileNode `
Not removing anything.

Reply:
We are no longer just reconciling the nodeRef, we get the node. This is why I changed the name as well.

Reply:
OK sounds good 👍